submissions/kazeemErinfolami/htmlForms/contact.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Contact</title>
+  </head>
+  <body>
+    <nav>
+      <ul>
+        <li><a href="../html/index.html">Home</a></li>
+      </ul>
+    </nav>
+    <form>
+      <div>
+        <h1>Contact Form</h1>
```

Comment:
header elements are block elements. You don't have to add it with a div if there is no extra use of the div for the header @kazeemerinfolami You can remove the div tags